ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml

```diff
@@ -78,7 +78,7 @@ Popup {
             appMain.changeAppSection(Constants.profile)
             // TODO: replace with shared store constant
             // Profile/RootStore.notifications_id
-            profileLayoutContainer.changeProfileSection(7)
+            Global.changeProfileSection(7)
```

Comment:
cna this be combined with the changeAppSection API? as a param? Just a thought for cleaner API. then in case we need to handle a use case for any other section, we won't have to write new methods.